src/content-helper/common/css/variables.scss

```diff
@@ -45,6 +47,7 @@ $base-unit-60: 3rem; // 48px.
 	--parsely-green-65: hsla(109, 59%, 26%, 1);
 	--gray-200: #f7f8f9;
 	--gray-300: #edeeef;
+	--gray-350: #e1e3e5;
```

Comment:
If this gray isn't part of dash's base.css, let's move it into the  additional variables section. We've got some colors there.

Reply:
I keep forgetting that these are based on the dash theme :) I moved it down to be on the same location as the other "custom" colors.